### ISSUE TITLE
Contibute to Emacs?

### ISSUE THREAD

**Issue by phil-s** [redguardtoo/diff-lisp#1]
In the spirit of `ls-lisp.el` which enables dired to pretend `ls` is available on MS Windows, this could potentially be useful on platforms (MS Windows, say) which do not come with `diff` as standard.

Have you considered contributing it, or discussing the notion on the `emacs-devel` list?

**Comment by redguardtoo** [comment]
Good idea.

**Comment by brongulus** [comment]
Hey, thanks for creating this great project. Have you thought about getting it into core? Or was there any discussion on `emacs-devel` regarding the same?

**Comment by redguardtoo** [comment]
Not sure. In Emacs core, algorithm is usually implemented in C  to get better performance.

**Comment by brongulus** [comment]
Understandable, it's unfortunate how there's no function in core to diff strings.

**Comment by phil-s** [comment]
I don't think that point should be seen as a deal-breaker -- if there's working lisp code, that can be useful *now*, and things can always be obsoleted in future if a faster alternative ever becomes available.  Consider `linum.el` -- added in 2008 and useful to many users for more than a decade before a version of Emacs with a C code approach was released.
